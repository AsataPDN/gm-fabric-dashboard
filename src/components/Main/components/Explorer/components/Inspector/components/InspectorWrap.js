import styled from "styled-components";

import { FONT_STACK_DATA, BORDER_RADIUS_BASE } from "style/styleVariables";

const InspectorWrap = styled.div`
  position: absolute;
  top: 0;
  right: 0;
  bottom: 0;
  left: 0;
  font-family: ${FONT_STACK_DATA};
  line-height: 1.2;
  height: 100%;
  border-radius: ${BORDER_RADIUS_BASE};
  display: flex;
  flex-direction: column;
`;

export default InspectorWrap;
