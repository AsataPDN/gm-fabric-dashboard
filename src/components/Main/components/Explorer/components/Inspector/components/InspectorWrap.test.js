import React from "react";
import { shallow } from "enzyme";
import InspectorWrap from "./InspectorWrap";

describe("InspectorWrap", () => {
  it("matches snapshot", () => {
    const aInspectorWrap = shallow(<InspectorWrap />);
    expect(aInspectorWrap).toMatchSnapshot();
  });
});
