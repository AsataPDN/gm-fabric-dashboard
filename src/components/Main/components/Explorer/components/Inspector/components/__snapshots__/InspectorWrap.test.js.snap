// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InspectorWrap matches snapshot 1`] = `
.c0 {
  border-radius: 3px;
  bottom: 0;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  font-family: Rubik,-apple-system,BlinkMacSystemFont,"Segoe UI","Roboto","Oxygen","Ubuntu","Cantarell","Fira Sans","Droid Sans","Helvetica Neue",sans-serif;
  height: 100%;
  left: 0;
  line-height: 1.2;
  position: absolute;
  right: 0;
  top: 0;
}

<div
  className="c0"
/>
`;
